entities/OfferType.php

```diff
@@ -40,9 +40,9 @@ public function __construct($connection){
 	public function create(){
 
 		if(is_admin()){
-			$sql = "INSERT INTO offer_types ( name,category_id,default_text,active,created_by,updated_by) values (:type,:name,:category_id,:default_text,:active,:user_id,:user_id)";
+			$sql = "INSERT INTO offer_types ( name,category,category_id,default_text,active,created_by,updated_by) values (:name,:category_name,:category_id,:default_text,:active,:user_id,:user_id)";
```

Comment:
We shouldn't be populating the category column. It shouldn't even exist any more. In production is is nullable, so I guess I didn't notice until now. We should remove the code populating it and drop the column

Reply:
We weren't sure why it was not removed when the primary key was switched to the ID in the first place.
But sure, we'll remove the category column.